### PR TITLE
trash: add page

### PR DESCRIPTION
**Pull request by Fealthas**
for trash-cli

https://github.com/andreafrancia/trash-cli

### CODE REVIEW

File: pages/linux/trash.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`trash-put {{path/to/file}}`
+
+- List files in trash:
+
+`trash-list`
+
+- Restore file from trash:
+
+`trash-restore`
```
Comment (schneiderl) [inline pages/linux/trash.md]: Doesn't that expect a parameter that says which is the file to be restored?

File: pages/linux/trash.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Empty trash:
+
+`trash-empty`
+
+- Empty trash older than {{X}} days:
```
Comment (schneiderl) [inline pages/linux/trash.md]: Maybe this can be rephrased. I think it is better to be clear that files on the trash that are older than X days will be deleted. With this description it feels like the trash bins will be deleted.

File: pages/linux/trash.md (hunk 1 above)
Comment (Fealthas) [reply pages/linux/trash.md]: Yes, but it seems kinda useless to me. 
Trash-list doesn't show the item id.
If you do not pass a parameter, it just lists all the files and waits for you to input a number, corresponding to the item to restore(actually shows numbers with trash-restore).

Comment (Fealthas) [reply pages/linux/trash.md]: Or actually no, it doesn't take parameters at all.

Comment (Fealthas) [reply pages/linux/trash.md]: Just running the command seems to be the only way to use it.

File: pages/linux/trash.md (hunk 2 above)
Comment (Fealthas) [reply pages/linux/trash.md]: Changed, it. Take a look.

File: pages/linux/trash.md (hunk 1 above)
Comment (schneiderl) [reply pages/linux/trash.md]: Oh thats good then. We can leave it as it is.

File: pages/linux/trash.md (hunk 2 above)
Comment (schneiderl) [reply pages/linux/trash.md]: Reads better now in my opinion. Thank you for your time 👍

File: pages/linux/trash.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`trash-restore`
+
+- Empty trash:
+
+`trash-empty`
+
+- Empty trash, keeping files trashed less than {{X}} days ago:
+
+`trash-empty {{X}}`
```
Comment (pxgamer) [inline pages/linux/trash.md]: Perhaps use a concrete example, such as `10`:
```suggestion
`trash-empty {{10}}`
```
And then update the description to:
`Empty trash, keeping files trashed less than 10 days ago:`

File: pages/linux/trash.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# trash
+
+> A CLI trashcan/recycling bin.
```
Comment (agnivade) [inline pages/linux/trash.md]: Perhaps - "A CLI for managing trash can / recycling bin" sounds better ?

Comment (schneiderl) [inline pages/linux/trash.md]: Add the homepage. 
You have alread linked it in your PR description. 
```suggestion
> A CLI trashcan/recycling bin.
> More information: <https://github.com/andreafrancia/trash-cli>.
```

File: pages/linux/trash.md (hunk 3 above)
Comment (Fealthas) [reply pages/linux/trash.md]: fixed

File: pages/linux/trash.md (hunk 4 above)
Comment (Fealthas) [reply pages/linux/trash.md]: fixed

**Comment by Fealthas** [comment]
Should I put just trash or trash-put?
Syntax checker seems to not like the "or".
I guess trash-put is unnecessary to know, as trash just links to trash-put IIRC, but it could be confusing if it is not documented.

**Comment by schneiderl** [comment]
@Fealthas As far as I understood they do the same thing. 
tldr is all about keeping it simple so i believe only one example should be listed. 

I would vote for "trash" because it looks clean.

File: pages/linux/trash.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# trash
+
+> A CLI for managing your trashcan / recycling bin.
+> More information: <https://github.com/andreafrancia/trash-cli>.
+
+- Delete a file (send to trash):
+
+`trash {{path/to/file}}`
+
+or
```
Comment (sbrl) [inline pages/linux/trash.md]: I think this is messing up the linter. Perhaps you could remove this?

While the `trash-put` might do the same thing, I'd probably suggest that it has it's own page, as generally speaking each command will have it's own page.

Comment (Fealthas) [reply pages/linux/trash.md]: fixed

**Comment by sbrl** [comment]
Sorry for the delay.

Are we ready to merge then, @schneiderl?

**Comment by schneiderl** [comment]
Sorry for the delay @Fealthas 
I have been a little busy the last few days.

Thank you for your contribution and feel welcome to keep contributing to tldr.
